crates/configuration/src/shared/node.rs

```diff
@@ -416,6 +447,8 @@ impl NodeConfigBuilder<Buildable> {
         )
     }
 
+    /// Set the P2P cert hash that will be used
+    // TODO: for what it can be used ?
```

Comment:
better description ?

Reply:
Set the P2P cert hash that will be used as part of the multiaddress, If and only if the multiaddress is set to use `webrtc`